frontend/components/molecules/MForgotPasswordDialog.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { FC } from 'react';
+import { Box, Dialog, IconButton, TextField, Grid, DialogContent, Button, FormControl } from '@material-ui/core';
+import { useState, ChangeEvent } from 'react';
+import { Close } from '@material-ui/icons';
+
+interface MForgotPasswordDialogProps {
+  showForgotPasswordModal?: boolean;
+  handleCloseForgotPasswordModal?: Function;
+}
+
+const MForgotPasswordDialog: FC<MForgotPasswordDialogProps> = ({
+  showForgotPasswordModal = false,
+  handleCloseForgotPasswordModal = () => {},
+}) => {
+  const [email, setEmail] = useState<String>('');
+
+  const handleEmailChange = (event: ChangeEvent<HTMLInputElement>) => {
+    setEmail(event.target.value);
+  };
+
+  const handleSubmit = () => {
+    alert(`You have entered ${email}`);
+    setEmail('');
+  };
+
+  return (
+    <Dialog open={showForgotPasswordModal} fullWidth>
+      <Box display="flex" justifyContent="flex-end" mt={2} mr={2}>
+        <IconButton aria-label="close" onClick={() => handleCloseForgotPasswordModal()} color="inherit">
```

Comment:
You can pass `handleCloseForgotPasswordModal` directly. 

This happens several times in other files, I won't write comments for those so it won't be too repetitive.